oxidation/crates/parsec_api_protocol/tests/test_block.rs

```diff
@@ -19,39 +19,105 @@ fn serde_block_create_req() {
         "7d7d4e95ad2fdea7b3bd19c5"
     );
 
-    let expected = BlockCreateReqSchema {
-        cmd: "block_create".to_owned(),
+    let req = authenticated_cmds::block_create::Req {
         block_id: "57c629b69d6c4abbaf651cafa46dbc93".parse().unwrap(),
         realm_id: "1d3353157d7d4e95ad2fdea7b3bd19c5".parse().unwrap(),
         block: b"foobar".to_vec(),
     };
 
-    let schema = BlockCreateReqSchema::load(&data).unwrap();
+    let expected = authenticated_cmds::AnyCmdReq::BlockCreate(req.clone());
+
+    let schema = authenticated_cmds::AnyCmdReq::loads(&data).unwrap();
```

Comment:
the name `schema` is not valid, should be `data` (especially given we have `data2` later on ^^)

This `schema` name is a bit everywhere in the tests. I suggest the following convention:
- `raw` for serialized stuff (this is already what we use in the codebase and python tests), `raw_xxx` if there is multiple raw elements in the test
- `data` for deserialized
- `req` for `authenticated_cmds::block_create::Req`
- `any_req` for `authenticated_cmds::AnyCmdReq`
- `expected` alone when the expected stuff is obvious, other wise `expected_xxx`

Reply:
Done :+1: